README.md

```diff
@@ -33,15 +33,15 @@ iex ((New-Object System.Net.WebClient).DownloadString('https://git.io/vanax'))
 ### Install/Configure/Start Puppet (Windows - production)
 ```shell
 # PowerShell
-$env:PuppetCollection = 'pc1'
+$env:PuppetCollection = '5'
```

Comment:
Can you add `$env:PuppetVersion = '5.5.22'`

Reply:
Sure

Reply:
Pushed, along with formatting the PS script.